tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -14741,6 +14741,11 @@ TEST_F(EnergyPlusFixture, Test_UnitarySystemModel_SubcoolReheatCoil)
     EXPECT_NEAR(thisSys->CoilSHR, thisSys->LoadSHR, 0.001);
     EXPECT_NEAR(SenOutput, -227.705, 0.1);
     EXPECT_NEAR(LatOutput, -1531, 0.1);
+    Real64 fanHeat = state->dataFans->Fan(thisSys->m_FanIndex).PowerLossToAir;
+    Real64 heatCoilOutput = state->dataHeatingCoils->HeatingCoil(thisSys->m_HeatingCoilIndex).HeatingCoilRate;
+    Real64 coilSensible = SenOutput - fanHeat - heatCoilOutput;
+    Real64 calcCoilSHR = coilSensible / (coilSensible + LatOutput);
+    EXPECT_NEAR(calcCoilSHR, thisSys->CoilSHR, 0.001);
```

Comment:
@rraustad Added a coilSHR check for each case in this unit test. These all fail. Maybe I didn't set this up right, or maybe, as you suspected, there's something wrong with `CoilSHR`.